modules/network-observability-resources-table.adoc

```diff
@@ -17,7 +17,7 @@ The examples outlined in the table demonstrate scenarios that are tailored to sp
 |                                     | Extra small (10 nodes) | Small (25 nodes)  | Medium (65 nodes) ^[2]^ | Large (120 nodes) ^[2]^
 | *Worker Node vCPU and memory*       | 4 vCPUs\| 16GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^
 | *LokiStack size*                    | `1x.extra-small`         | `1x.small`          | `1x.small`           | `1x.medium`
-| *Network Observability controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi
+| *Network Observability controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 400Mi
 | *eBPF sampling rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
 | *eBPF memory limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default)
```

Comment:
```suggestion
| *eBPF memory limit*                 | 800Mi (default)        | 800Mi (default)   | 800Mi (default)             | 1600Mi
```
@skrthomas might have messed up the spacing here but these are the new numbers :smile:

Reply:
@nathan-weinberg Thanks. Addressed!